spec/support/shared_examples/touch_course_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
+
+shared_examples 'Touch course' do
+  describe '#touch_course' do
+    let!(:model) { create(described_class.model_name.singular.to_sym) }
```

Comment:
This feels like a potential gotcha. Would passing in an object as a parameter to this shared example be worse?

Reply:
Yeah agreed, that would be easier to understand!